##Still I able to manage to call from external API but failed to execute try block..INTERNAL SERVER ERROR
>>npm i lodash
>>
>>npm i axios
>>
>>npm i cors
